releases/2021.1/2022.1.md

```diff
@@ -0,0 +1,165 @@
+
+**mef_eline**:
+
+- Bug fixes and new features UI features that are described in subsection 4
+- Added new event to allow `available_tags` synchronization on `topology`
+- Parametrized `force` option when sending request to `flow_manager` leveraging this new feature
+- Concurrency bug fixes
+
+**flow_manager**:
+
+- Consistency check improvements and optimizations, faster indexed lookups, [EP24 blueprint](https://github.com/kytos-ng/kytos/blob/master/docs/blueprints/EP024.rst)
+- Added support for OpenFlow 1.3 message types of `OFPT_BARRIER_REQUEST` and `OFPT_BARRIER_REPLY`
+- New event to improve overall OpenFlow FlowMod messages and state management, including handling connection errors
+- Added support for the `force` option when sending OpenFlow FlowMod messages
+- Concurrency bug fixes
+
+**topology**:
+
+- Bug fixes handling port status changes for interfaces and links
+- Added feature to store link `available_tags`
+- Optimized and minimized events for Link up notifications
+- Concurrency bug fixes
+
+**of_core**:
+
+- Fixed `handle_raw_in` to avoid data loss under high usage scenarios
+- Implemented support to add a new action class, which enable the use of experimenter action
+- Fixed `switch.lastseen` when handling OpenFlow features reply
+- Changed `update_port_status` on `OFPPR_DELETE` to deactivate an interface instead of deleting to keep `topology` consistent
+- Concurrency bug fixes
+
+**noviflow**:
+
+- Added support to NoviFlow OpenFlow custom experimenter actions to enable INT and BFD
+
+**pathfinder**:
+
+- Added support computing constrained best paths, [EP23-2 blueprint](https://github.com/kytos-ng/kytos/blob/master/docs/blueprints/EP023-2.rst)
+- Added Web UI components to be able to search with the new constrained path attributes
+- Fixed bug edge case when building the topology graph with admin down interfaces and links
+- Added support to update graph based on Link metadata events
+
+
+### 2. Deploy Kytos-ng and NApps on testing and production-like environments
+
+The following switches have been with `kytos` version `2022.1`:
+
+Switch Vendor                        | Switch          | Firmware version        | OpenFlow version | Environment type
+-------------------------------------|-----------------|-------------------------|------------------|-----------------
+[OVS](https://www.openvswitch.org/)  | ovs-switchd     | 2.15                    | 1.3              | testing
+[NoviFlow](https://www.noviflow.com) | WB5132-F        | NW570.2.1 and NW550.2.3 | 1.3              | production 
+
+[kytos-end-to-end-tests](https://github.com/amlight/kytos-end-to-end-tests) suite was only run for OVS in this release. In the future, other switch target platforms might be supported by this test suite.
+
+### 3. Perform network stress terms to verify performance, reliability an overall stability
+
+**TODO - Ask if Italo can also add more quantitative data here based on the tests he ran and presented**
```

Comment:
@italovalcy, amazing information. Good stuff. One question though:

Was it 48 hours or you meant another unit here? If it was too long we it'd be great to have an issue for our team to look into it later. 

>  Otherwise, some requests would last more than 48hs with no response.

Reply:
> @italovalcy, Amazing information, and impressive results. Good stuff. One question though:
> 
> Was it 48 hours or you meant another unit here? If it was too long it'd be great to have an issue for our team to look into it later (if I don't have one already that I've missed).
> 
> > Otherwise, some requests would last more than 48hs with no response.

Thank you very much @viniarck! Regarding the 48h issue, it was actually in the previous version of Kytos 2021.1. I'll review the text to make it more clear. Thanks for the suggestion.